Feature/m0081 meja 1638 document portable fo (#375)
MicroEJ/docs: KernelDeveloperGuide/applicationLink.rst

   - Building the virtualization tables.

>    The result is an object file that ends with ``.fo`` extension.

Reviewer: ```suggestion
   The result is an object file that ends with the ``.fo`` extension.
```

>    By default, the ``.fo`` file is specific to a Kernel: it can only be installed on the Kernel it has been linked. 

Reviewer: ```suggestion
   By default, the ``.fo`` file is specific to a Kernel: it can only be installed on the Kernel it has been linked to. 
```

>    Rebuilding a Kernel implies to run this phase again, unless it has been built from the previous Kernel (see :ref:`feature_portability`).   

Reviewer: ```suggestion
   Rebuilding a Kernel implies to run this phase again, unless the application has been built for the previous Kernel (see :ref:`feature_portability`).   
```

.. [#compatible_def] New version is greater than or equals to the old one within the same major version.

.. _feature_portability:

Feature Portability
-------------------

By default, a ``.fo`` file can only be installed on the Kernel on which it has been linked. 

> Starting from Architecture version ``8.0.0``, a Feature can be installed on different Kernels. This is called `Feature Portability`.

Reviewer: ```suggestion
Starting from Architecture version ``8.0.0``, the same Feature file can be installed on different Kernels. This is called `Feature Portability`.
```
